charts/argo-workflows/templates/server/server-ingress.yaml

```
@@ -0,0 +1,39 @@
+{{- if .Values.server.enabled -}}
+{{- if .Values.server.ingress.enabled -}}
+{{- $serviceName := include "argo-workflows.server.fullname" . -}}
+{{- $servicePort := .Values.server.servicePort -}}
+apiVersion: {{ include "argo-workflows.ingress.apiVersion" . }}
+metadata:
+  name: {{ template "argo-workflows.server.fullname" . }}
+  labels:
+    {{- include "argo-workflows.labels" $ | nindent 4 }}
```

```suggestion
   {{- include "argo-workflows.labels" (dict "context" . "component" .Values.server.name "name" .Values.server.name) | nindent 4 }}
```

Making this same as other references to this template. Was getting an error otherwise.
error calling include: template: argo-workflows/templates/_helpers.tpl:57:39: executing \"argo-`workflows.selectorLabels\" at <.context.Release.Name>: nil pointer evaluating interface {}.Release"`